documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+ 
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/console`)
+       services:
+       - kind: Service
+         name: domain1-adminserver
+         namespace: weblogic-domain
+         port: 7001
+   ```
+
+2. Open the following URL from your browser to access the WebLogic Server Administration Console:
+
+   ```
+   http://${HOSTNAME}:${LB_PORT}/console
+   ```
+   Where:
+   
+     * `${HOSTNAME}` is where the ingress load balancer is running.
+   
+     * To determine the `${LB_PORT}` when using a Traefik load balancer:
+
+        `$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')`
+
+   If you have an [FMW Infrastructure]({{< relref "/managing-fmw-domains.md" >}}) type domain, then you can add an ingress path routing rule for PathPrefix `/em` and access the Fusion Middleware Control (Enterprise Manager) Console using the following URL:
+
+   ```
+   http://${HOSTNAME}:${LB_PORT}/em
+   ```
+#### Access the WebLogic Server Administration Console through the SSL port 
+1. Enable `WebLogic Plugin Enabled` on the WebLogic domain level
+
+   If you are using WDT to configure the WebLogic domain, you need to add the following resource section at the domain level to the model YAML file.
+   ```yaml
+   resources:
+        WebAppContainer:
+            WeblogicPluginEnabled: true
+   ```
+   If you are using a WLST script to configure the domain, then the following modifications are needed to the respective WLST python script.
+   ```javascript
+   # Configure the Administration Server
+   cd('/Servers/AdminServer')
+   set('WeblogicPluginEnabled',true)
+   ...
+   cd('/Clusters/%s' % cluster_name)
+   set('WeblogicPluginEnabled',true)
+   ```
+2. Configure an ingress path routing rule and update the ingress resource with customRequestHeaders value
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer. In case of SSL termination, Traefik must pass a custom header `WL-Proxy-SSL:true` to the WebLogic Server endpoints. Following example creates the Traefik Middleware custom resource with the custom request header `WL-Proxy-SSL:true`. If you have multiple domains managed by a single ingress controller, then see the [Host-based secured routing](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/charts/traefik/README.md#host-based-secured-routing) example or consider using the [Remote Console]({{< relref "/managing-domains/accessing-the-domain/admin-console/_index.md">}}).
```

Comment:
The "If you" sentence can be moved to a common area for the two Ingress sections (non-SSL and SSL).

Reply:
Done.